spec/rails_helper.rb

```diff
@@ -119,108 +97,49 @@ def stub_addresses
   # Make FactoryBot easier.
   config.include FactoryBot::Syntax::Methods
 
-  # Stub out Geocoder or else...
-  config.before(:all) do
-    stub_addresses
-  end
-
-  # set driver for system tests
-  config.before(:each, type: :system) do
-    clear_downloads
-    driven_by :chrome
-    Capybara.server = :puma, { Silent: true }
-  end
-
-  config.after(:each, type: :system, js: true) do
-    clear_downloads
-  end
-
   # Preparatifyication
   config.before(:suite) do
```

Comment:
This approach *should* work - one thing to keep in mind is that if any specs need to break out of transactions (e.g. by testing something that has an `after_commit` hook) they'd have to manually clean via truncation and re-insert whatever is supposed to go in there afterwards. I've been bitten by that before. Not sure if there's anything we can do that about that in advance though.

Reply:
Good question. I don't know of any other yet, but I'd actually question the use of `after_commit` and see if it makes sense. I think for those situations we might be able to add something ontop of what we have.